templates/partials/summary/collapsible-summary.html

```diff
@@ -2,6 +2,10 @@
 {% from "components/summary/_macro.njk" import onsSummary %}
 
 {%- set itemList = [] -%}
+{%- set tableHeader = ["Question", "Answer given"] -%}
+{%- if content.summary.answers_are_editable -%}
+  {{ tableHeader.append( "Change answer" ) or "" }}
```

Comment:
Why is the `or ""` needed, the functional tests appear to pass without it? Is there a test scenario where this is needed that we need to add?

Reply:
Its needed because when there is a change link Jinja will print out "None" like this:

![Screenshot 2022-08-08 at 08 49 29](https://user-images.githubusercontent.com/42928680/183367502-7e35a13f-0bf6-4a55-be5d-73d5ae0e78c6.png)

This is because strings are truthy in Jinja, and `None` is falsy, adding the `or ""` will return the empty string and then output nothing on the page

Reply:
I know this logic is in the backend now, but when I remove `else: headers.extend("")` from the code, I don't see this `None` when I get to a summary page

Reply:
Which schema did you use to test this cause I can't get the `None` to appear when I remove the else?

Reply:
So this means we can remove:
`else:
            headers.extend("")`?

Reply:
Yeah true, ill remove that 👍